src/azure-cli/azure/cli/command_modules/signalr/custom.py

```diff
@@ -13,15 +13,16 @@
 
 
 def signalr_create(client, signalr_name, resource_group_name,
-                   sku, unit_count=1, location=None, tags=None, service_mode='Default', allowed_origins=None, default_action="Allow"):
+                   sku, unit_count=1, location=None, tags=None, service_mode='Default', enable_message_logs='False', allowed_origins=None, default_action="Allow"):
```

Comment:
When using `get_three_state_flag`, the parameter should be a bool. Also, user should use the lowercase `true`/`false` to keep the convention.